src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$daySize: 100px;
+$countDays: 31;
+$gapValue: 1px;
+$calendarWidth: $daySize * 7 + 10;
```

Comment:
What does 7 and 10 mean in this formula?
Looks like magic nums, better to use vars here